### ISSUE TITLE
Heat bed changes temperature to match support interface layer

### ISSUE THREAD

**Issue by camfarris** [bambulab/BambuStudio#3275]
### Bambu Studio Version

1.8.2.56

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 11

### Printer

Bambu Lab P1S

### How to reproduce

1. Slice print with supports where base and interface layer are 2 different materials (example: PETG base and PLA interface layer)
2. Begin print
3. Wait for print to start an interface layer

### Actual results

When swapping to interface layer, both the nozzle and heat bed change to the interface layer material's temperature instead of the base temperature

### Expected results

Only the nozzle temperature changes to reflect the interface layer material's temperature. The heat bed remains the base material temperature.

### Project file & Debug log uploads

Behavioral expectation issue

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by XunZhangBambu** [comment]
Hello,you mean do not change heat bed temperature while switching to print interface layer with a different filament ?

**Comment by camfarris** [comment]
@XunZhangBambu 

Yes. The heat bed should not change when swapping to the interface layer. This is because the interface layer does not touch the heat bed. The temperature should match the base material temperature.

**Comment by Schokobecher** [comment]
Current work around is to make a material profile with the same bed temperature. But I agree, it should be at least a toggle/dropdown for the behavior.

**Comment by scobby** [comment]
Hey there, having the exact same behavior. Only solutions for me was to set the pla filament bed temperature temporary for the print to the temperature of the PETG.
Based on the youtube videos around the topic how to get perfect supports with PLA/PETG i thing this is a commen problem for many people why prints fail in this combination. And its hard to track the problem to the cause.
Could you please fix this in the next release? @XunZhangBambu

**Comment by david-breidert** [comment]
Commenting to bump this issue. This is still happening and with the release of the H2D even more important to fix.

**Comment by mean-ui-thread** [comment]
I don't know for certain until I experiment a bit more, but every time I use Bambu support material with PETG filament, my layers get misaligned on the same layers than the support/raft interface layers. If I disable support, then my PETG prints are fine (except for the overhangs obviously)

My custom PETG filament setting sets the bed temperature to 77C, and Bambu support filament drops it to 60C, which is most likely the root cause of my print issue I believe.

![Image](https://github.com/user-attachments/assets/8ec14bab-e9b9-4e1f-95e5-61b82243e918)

![Image](https://github.com/user-attachments/assets/72e0c77e-124e-47d5-8f38-8a17a8b7b718)

![Image](https://github.com/user-attachments/assets/4d418697-dfe9-4676-b351-0833fadb5acc)

For now, I will manually tweak the bed temperature of Bambu support filament to ensure that the bed temperature doesn't drop, and will do some tests to see if that solves the issue. If the resolves the issue, than yes, I agree 100%. the support filament should not modify the bed temperature. This should be automatic.

Bambu is advertising their support filament for both PLA and PETG but its settings are only for PLA prints. They really dropped the ball on that one.

**Comment by LouisKleiman** [comment]
This is still a problem in version 2.7.1.62.  Come on, guys. BL creates a smooth experience, right? Two and a half years and this still isn't corrected?  I am using HIPS as an interface layer for a PBT print.  Bed temp for HIPS is 90, for PBT is 65.  First, I have to create a custom H2D printer and change it to set the bed temperature by First Filament, not Max.  Once that is done, I get the right temp (65) initially, but at the first interface layer, the bed temp increases to 90.  Apparently the only way to fix that is to change the HIPS settings, create a new HIPS variant, whatever....  This problem makes the Printer - Bed temperature type setting useless.  Your test cases are completely insufficient here.
